### ISSUE TITLE
 Synchronising  not completing

### ISSUE THREAD

**Issue by SureshCBE** [mozilla-lockwise/lockwise-android#1252]
Hi,
Synchronising not complete even after one hour.

**Comment by ameaninglessname** [comment]
Is your network connection stable enough?

The syncing experience is definitely not good.
It's really slow and not responsive sometimes.

But finally, it would always work after re-tried for several times.

**Comment by jasondeavenport** [comment]
Jason deavenport

**Comment by SureshCBE** [comment]
Yes my internet connection is stable.On 04-Feb-2021 21:53, Incredible <notifications@github.com> wrote:
Is your network connection stable enough?
The syncing experience is definitely not good.
It's really slow and not responsive sometimes.
But finally, it would always work after re-tried for several times.

**Comment by bunnybooboo** [comment]
Replicated Lineage OS version 18.1 / Android version 11 and with application uninstall / re-install. Lockwise Beta version 4.0.3.

**Comment by ooksanen** [comment]
Also on OxygenOs 11 / Android 11. Lockwise 4.0.3 (6584)

Edit: Uninstall / re-install fixed the problem. At least for now,

**Comment by KarolStosik** [comment]
I have the same on Android 11. Synchronization never ends. If I kill the app and start again, it'll be ok.
